pubspec.yaml

```diff
@@ -76,6 +76,7 @@ dependencies:
   sprintf: ^6.0.0
   flutter_localizations:
     sdk: flutter
+  intl_utils: ^2.4.0
```

Comment:
we shoud remove the `flutter pub add intl_utils` from the workflow file, otherwise the ci will fail